pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go

```diff
@@ -1054,3 +1084,19 @@ func (gqm *GroupQuotaManager) doUpdateOneGroupSharedWeightNoLock(quotaName strin
 
 	gqm.updateOneGroupSharedWeightNoLock(quotaInfo)
 }
+
+func isTerminatingPod(pod *v1.Pod) bool {
```

Comment:
It is strange to add feature-gate in isTerminatingPod. Why not change the name of this function to shouldBeIgnored and also change the previous related functions.

Reply:
fix it